python/shotgun_model/simple_shotgun_model.py

```diff
@@ -70,6 +70,7 @@ def load_data(
                   is handling, allowing a user to for example show the twenty most recent notes or
                   similar.
         :param columns: List of Shotgun fields to use to populate the model columns
+        :param editable_columns: A subset of ``columns`` that will be editable in views that use this model.
         :param additional_filter_presets: List of Shotgun filter presets to apply, e.g.
```

Comment:
would be great with some additional info about this additional_filter_presets stuff too - it feel reall mysterious! Is there a reference where i can go look and see what the heck `BY_PIPELINE_STEP_NUMBER_AND_ENTITIES_CREATED_AT` means and what other things are available?